src/functions/atime.h

```diff
@@ -48,7 +47,7 @@ inline const QString minutesToString(int minutes_in, opl::time::FlightTimeFormat
     {
         int hour = minutes_in / 60;
         double minute = (minutes_in % 60) / 60.0;
-        return QString::number(hour+minute);
+        return QString::asprintf("%.2f", (hour+minute));
```

Comment:
Qt recommends arg for future code. https://doc.qt.io/qt-5/qstring.html#asprintf.
```c++
return QString::number("%1").arg(hour + minute, 0, 'f', 2);
```

Reply:
Good idea, on it!